core/src/main/scala/scalaz/Applicative.scala

```diff
@@ -131,5 +131,22 @@ object Applicative {
 
   implicit def monoidApplicative[M:Monoid]: Applicative[λ[α => M]] = Monoid[M].applicative
 
+  implicit def eitherTParallelApplicative[F[_], E](implicit F0: Applicative.Par[F]): Applicative.Par[EitherT[F, E, ?]] =
```

Comment:
please define in `EitherT.scala`

Reply:
It doesn't work  :disappointed: 

The reason is:
```scala
type Par[F[_]] = Applicative[λ[α => F[α] @@ Tags.Parallel]]
type @@[T, Tag] = Tagged[T, Tag]
private[scalaz] type Tagged[A, T] = {type Tag = T; type Self = A}
// =>
Applicative.Par[EitherT[F, E, ?]] = Applicative[λ[α => {type Tag = Parallel; type Self = EitherT[F, E, α]}]]
```

But type member (`Self`) is not in implicit scope and therefore `EitherT` is not in implicit scope.

Reply:
@xuwei-k see #2076

Reply:
I think I wasn't quite clear, #2076 would enable to move the instance to `EitherT`